packages/studio-ui/src/components/PropEditor.tsx

```diff
@@ -48,9 +49,18 @@ export default function PropEditor({
     <div className="flex mb-2 text-sm grow flex-col">
       <div className="flex relative">
         {isNested && renderBranchUI()}
-        <label className="pb-1" id={labelTooltipId} htmlFor={inputId}>
-          {propName}
-        </label>
+        <div className="flex flex-row">
+          <label className="pb-1" htmlFor={inputId}>
+            {propName}
+          </label>
+          {tooltip && (
+            <Info
```

Comment:
What do you think of separating out this Info icon and the actual tooltip into a common component? That would make it easier to reuse if we decide we want this icon for tooltips in other places in the Studio UI